tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -32,6 +35,21 @@ EOF
 	ln -s /run "$rootfs_dir/var/run"
 	cp --remove-destination /etc/resolv.conf "$rootfs_dir/etc"
 
+    if [ "${TEE_PLATFORM}" == "tdx" ] && [ "${ARCH}" == "x86_64" ]; then
+        source /etc/os-release
+        if [ "${VERSION_ID}" == "20.04" ]; then
+            curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key |  chroot "${rootfs_dir}" apt-key add -
+
+            echo 'deb [arch=amd64] http://security.ubuntu.com/ubuntu focal-security main universe' | tee ${rootfs_dir}/etc/apt/sources.list.d/universe.list
+            echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | tee ${rootfs_dir}/etc/apt/sources.list.d/intel-sgx.list
+            chroot "${rootfs_dir}" apt-get update && chroot "${rootfs_dir}" apt-get install -y libtdx-attest=1.15\* libtdx-attest-dev=1.15\*
```

Comment:
main branch TDX kernel upgraded to 6.2.x, dcap also need upgrade to 1.18

Reply:
fixed, thanks